autorest/codegen/serializers/general_serializer.py

```diff
@@ -18,7 +18,10 @@ def config_imports(code_model, global_parameters: ParameterList, async_mode: boo
     for gp in global_parameters:
         file_import.merge(gp.imports())
     if code_model.options["azure_arm"]:
-        file_import.add_from_import("azure.mgmt.core.policies", "ARMHttpLoggingPolicy", ImportType.AZURECORE)
+        async_policy = "ARMHttpLoggingPolicy, AsyncARMChallengeAuthenticationPolicy"
```

Comment:
One thing to note is that the base `azure-mgmt-core` version in the `basic-setup-py` needs to be updated to the `azure-mgmt-core` version that added `ARMChallengeAuthenticationPolicy`. Please update [here](https://github.com/Azure/autorest.python/blob/autorestv3/autorest/codegen/templates/setup.py.jinja2. At the same time, can you update the minimum `azure-mgmt-core` req in [this doc](https://github.com/Azure/autorest.python/blob/autorestv3/docs/client/initializing.md#minimum-dependencies-of-your-client)?

Reply:
sure. `setup.py.jinjia2` has been updated. And I will update the [this doc](https://github.com/Azure/autorest.python/blob/autorestv3/docs/client/initializing.md#minimum-dependencies-of-your-client) after new autorest.python is published.

Reply:
sounds good, it's also possible to update in this PR if you want to keep everything in one PR